spec.html

```diff
@@ -25970,33 +25974,26 @@ <h1>Object.freeze ( _O_ )</h1>
         </emu-alg>
       </emu-clause>
 
-      <emu-clause id="sec-object.fromentries">
+      <emu-clause id="sec-object.fromentries" oldids="sec-create-data-property-on-object-functions">
         <h1>Object.fromEntries ( _iterable_ )</h1>
         <p>When the `fromEntries` method is called with argument _iterable_, the following steps are taken:</p>
         <emu-alg>
           1. Perform ? RequireObjectCoercible(_iterable_).
           1. Let _obj_ be OrdinaryObjectCreate(%Object.prototype%).
           1. Assert: _obj_ is an extensible ordinary object with no own properties.
-          1. Let _stepsDefine_ be the algorithm steps defined in <emu-xref href="#sec-create-data-property-on-object-functions" title></emu-xref>.
-          1. Let _adder_ be ! CreateBuiltinFunction(_stepsDefine_, &laquo; &raquo;).
-          1. Return ? AddEntriesFromIterable(_obj_, _iterable_, _adder_).
-        </emu-alg>
-        <emu-note>
-          The function created for _adder_ is never directly accessible to ECMAScript code.
-        </emu-note>
-
-        <emu-clause id="sec-create-data-property-on-object-functions">
-          <h1>CreateDataPropertyOnObject Functions</h1>
-          <p>A CreateDataPropertyOnObject function is an anonymous built-in function. When a CreateDataPropertyOnObject function is called with arguments _key_ and _value_, the following steps are taken:</p>
-          <emu-alg>
+          1. Let _closure_ be a new Abstract Closure with parameters (_key_, _value_) that performs the following steps when called:
             1. Let _O_ be the *this* value.
```

Comment:
I'm not sure it makes sense for an abstract closure to access `the *this* value`. Anyway, I think it would be clearer for the closure to capture `_obj_` and use that instead of `_O_`.